xarray/tests/test_dask.py

```diff
@@ -292,6 +293,23 @@ def test_persist(self):
         self.assertLazyAndAllClose(u + 1, v)
         self.assertLazyAndAllClose(u + 1, v2)
 
+    # Need Pint 0.15 for __dask_tokenize__ implementation
+    @requires_pint_0_15
```

Comment:
I was about to propose extending `_importorskip` when I realized that you can, in fact, do
```suggestion
    @requires_pint_0_15(reason="needs __dask_tokenize__")
```
and the test will still be skipped if the required package is not available.